UnitTests/GitCommandsTests/ArgumentBuilderExtensionsTests.cs

```diff
@@ -263,7 +263,7 @@ public void Add_ForcePushOptions_ensure_all_switches_are_handled()
             foreach (int mode in Enum.GetValues(typeof(ForcePushOptions)))
             {
                 // unimplemented switches will result in InvalidEnumArgumentException
-                var args = new ArgumentBuilder
+                new ArgumentBuilder
```

Comment:
💡 Consider using a discard so it's clear to the reader that this is not accidental

```
_ = new ArgumentBuilder
```